src/common/storage/tests/it/column_leaf.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Datafuse Labs.
```

Comment:
`ColumnLeaf` is renamed to `ColumnNode` in #9660. So the file name should be renamed later.